<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FriendListWindow</class>
 <widget class="QDialog" name="FriendListWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>550</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>300</width>
    <height>550</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>300</width>
    <height>550</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QListWidget" name="friendListWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>300</width>
     <height>500</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>510</y>
     <width>211</width>
     <height>26</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="addFriendButton">
      <property name="text">
       <string>ADD</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="removeFriendButton">
      <property name="text">
       <string>REMOVE</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
